packages/components/src/components/telekom/logo/logo.tsx

```diff
@@ -52,6 +52,7 @@ export class Logo {
   /** (optional) Hide all logo related titles */
   @Prop() logoHideTitle?: boolean;
   @Prop() logoAriaDescribedBy?: string;
+  @Prop() logoAriaHide?: boolean = false;
```

Comment:
Wouldn't you call this one `logoAriaHidden` following the other similar "inner aria" props?